src/main/java/org/fenixedu/learning/domain/executionCourse/ExecutionCourseSite.java

```diff
@@ -56,13 +58,16 @@
     public ExecutionCourseSite(ExecutionCourse executionCourse) {
         checkNotNull(executionCourse);
         setExecutionCourse(executionCourse);
-        setPublished(true);
+
         setFolder(folderForPath(PortalConfiguration.getInstance().getMenu(), "courses"));
         setSlug(on("-").join(getExecutionCourse().getSigla(), getExecutionCourse().getExternalId()));
-        setCanAdminGroup(NobodyGroup.get());
-        setCanPostGroup(NobodyGroup.get());
-        setBennu(Bennu.getInstance());
 
+        setCreationDate(new DateTime());
+        setCanAdminGroup(DynamicGroup.get("managers"));
+        setCanPostGroup(TeacherResponsibleOfExecutionCourseGroup.get(executionCourse));
```

Comment:
I agree, for now lets put it at Nobody, and when we have the access control on CMS revisit this problem. What I would suggest doing on your installations is setting this to the appropriate access control using signals or scripts.